### ISSUE TITLE
[Desktop] New welcome to Pro screen for unauthorized Pro user

### ISSUE THREAD

**Issue by Derekf5** [getlantern/lantern#4855]
This screen will pop up when a free users friends have checked out with their referral code similar to the way the current "welcome to pro" screen pops up after checkout.

The difference will be that this screen will force the user to input email and authorization code, eliminating the need for users to do this later using the unauthorized pro account management screen.

This was born out of ticket #4789

**Comment by sunshine19090** [comment]
Is this going to be in MVP @Derekf5 ?

**Comment by Derekf5** [comment]
not sure @sunshine19090  its up to @myleshorton

**Comment by Derekf5** [comment]
@sunshine19090 just noting what @myleshorton this will not be for the MVP

**Comment by carsonefv** [comment]
Here are the two pop-up screens that should appear one after another when a free user gets free months of Pro after a friend has checked out with their referral code. They use a similar format to the "Welcome to Pro" screen that appears after checkout CC: @fffw @myleshorton

**1.**
![screenshot 2016-06-30 12 50 28](https://cloud.githubusercontent.com/assets/4400763/16496714/4fb78a22-3ec1-11e6-8a1b-50957ed17ffb.png)

• Text: "Congratulations! Your Referral Code was used by a friend to buy Lantern Pro. X Free months of Pro have been applied to your account, confirm your email to start using Lantern Pro."
**Font: Roboto Medium (500); Font Size: 14pt; Line-height: 24pt; color: #000000; align: center;**

• Email entry section text is same as "authorize device" screens with adapted padding

**2.**
![screenshot 2016-06-30 13 22 08](https://cloud.githubusercontent.com/assets/4400763/16497580/b0d05e7a-3ec5-11e6-9957-e6e99a67cc56.png)

• Change Top Bar icon to "Back Arrow" (same as arrow in checkout flow promotion screen)
• Change Top Bar Text to "Authorize Device"
• Code entry section is same as "authorize device" screens with edited text

**Comment by sunshine19090** [comment]
@Derekf5 @carsonefv @myleshorton There is a possibility that the user might close the screen and just proceed to pro. In that case, the user will never be able to verify their email address. It will be better if we remove the close button in these screens and do not let the user to continue until they verify their email. If we do keep the close button, we should probably provide a link in account management screen to verify their email in place of email address. 

I also feel that we are making this complicated.
I have two suggestions - 
1. We can show just the email verification screen and inform the user that they can proceed to pro after verifying. 
2. Let the user verify the email address whenever they wanted to. We can just provide a link in the main page saying "Please verify your email address. Click here" or something that will inform the user to verify email address in order to authorize other devices.

**Comment by Derekf5** [comment]
I agree with that @carsonefv .   From a user perspective i think this makes linking devices much easier, so i am in favor of removing the close button.

**Comment by sunshine19090** [comment]
Makes sense @Derekf5 .

**Comment by myleshorton** [comment]
Definitely not MVP @sunshine19090 and @Derekf5. Just for some context and for the future, typically changes like this wouldn't even be added even if they were proposed a week or so ago, so definitely the day before is cutting it way too close for something like this. Basically be the last week before a big release we should be purely working on bug fixing and not adding any new functionality except for in really exceptional circumstances.

**Comment by Derekf5** [comment]
Cool thanks what I figured @myleshorton and wasn't expecting it in this build.   I just wanted to get this design fix into github since it seemed to be an area that was confusing the team all around.

**Comment by Derekf5** [comment]
closing b/c no longer relevant

**Comment by Derekf5** [comment]
Hey @myleshorton this is the ticket we spoke about, i think we just need the first screen from above.   what do you think?
![image](https://cloud.githubusercontent.com/assets/7864729/18210735/01ce8d64-7108-11e6-815b-9e15c61979f8.png)

**Comment by p3rcya** [comment]
@Derekf5 Do we have to confirm email via an email code? The email might be blocked. Maybe it's better to have email and confirm email.

**Comment by Derekf5** [comment]
yeah thats why i mention above to take out that step

**Comment by Derekf5** [comment]
we could add a confirm field as well @p3rcya

**Comment by p3rcya** [comment]
Yeah, I think it should just be email, confirm email, but not sending an email to users to confirm.

**Comment by songxuangh** [comment]
## 输入我的邀请码（Enter my invitation code）

> # GY6S6C

### 来获得三个月的蓝灯专业版（To get three months of Lantern Professional Edition）
